src/frontend/components/ProductList/ProductList.tsx

```diff
@@ -12,11 +12,11 @@ interface IProps {
 
 const ProductList = ({ productList }: IProps) => {
   return (
-    <S.ProductList data-cy={CypressFields.ProductList}>
-      {productList.map(product => (
-        <ProductCard key={product.id} product={product} />
-      ))}
-    </S.ProductList>
+      <S.ProductList data-cy={CypressFields.ProductList}>
+        {productList.map(product => (
+            <ProductCard key={product.id} product={product} />
+        ))}
+      </S.ProductList>
```

Comment:
Is this change needed?

Reply:
I tried to remove this change, I thought I succeeded :D thanks for the catch. 
(historically I've put the provider around this element, but then the product page would throw an error, so I moved it to root level)